src/pattern-library/components/patterns/prototype/SelectPage.tsx

```diff
@@ -447,6 +447,40 @@ export default function SelectPage() {
               </Library.InfoItem>
             </Library.Info>
           </Library.Example>
+          <Library.Example title="listboxOverflow">
+            <Library.Info>
+              <Library.InfoItem label="description">
+                Determines the behavior of the listbox options when their
```

Comment:
```suggestion
                Determines the behavior of the listbox options when their
```

"options" or "items"? The component is called `Select.Option` but code examples tend to use the term "items" for the data arrays from which the options are generated.

Reply:
I think I would use "items" when conceptually talking about the entries of the list, and "options" when talking about the markup elements explicitly.

In this case, since we are talking about how they render, I think I would use "options".

But this is not a strong opinion, and I see a lot of grey areas in that reasoning, so I'm ok using a different wording.

Reply:
> I think I would use "items" when conceptually talking about the entries of the list, and "options" when talking about the markup elements explicitly.

That makes sense.